docs/siem/detections/api/lists/api-delete-list-item.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[lists-api-delete-item]]
+=== Delete list item
+
+Deletes a list item.
+
+==== Request URL
+
+`DELETE <kibana host>:<port>/api/lists/items`
+
+===== URL query parameters
+
+The URL query must include one of the following:
+
+* `id` - `DELETE /api/lists/items?id=<id>`
+* `list_id` and `value` - `DELETE /api/lists/items?list_id=<list_id>&value=<value>`
+
+===== Example request
+
+Deletes the list item with an `id` of `internal-ip-london`:
+
+[source,console]
+--------------------------------------------------
+DELETE api/lists/items?id=internal-ip-london
+--------------------------------------------------
+// KIBANA
+
+==== Response code
+
```

Comment:
Technically this will return the body of the thing it just deleted. Not all our REST endpoints do from Kibana but I typically do within the ones I write so the user knows not just that they deleted something but that they did delete the item they were expecting. Optional if you want to show that or not.